schemas/calendar-v1.0.0.json

```diff
@@ -0,0 +1,473 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/rayners/fvtt-seasons-and-stars/schemas/calendar-v1.0.0.json",
+  "title": "Seasons & Stars Calendar Schema",
+  "description": "JSON schema for Seasons & Stars calendar format v1.0.0",
+  "type": "object",
+  "required": ["id", "translations", "year", "months", "weekdays", "time"],
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": "string",
+      "pattern": "^[a-z0-9-]+$",
```

Comment:
Add `;` and `/`, maybe some other special characters too. I want people to introduce their own prefix for calendar names within a group or something, like all my calendars could have ids like "rayners/calendar-1", "rayners/calendar-2" and similar. Significantly reduces the likelihood of id collisions.

Reply:
Namespacing! The word I was looking for is namepsacing.